File: perl/fizzbuzz.pl
for ($i = 1; $i <= 30; $i++) {
  my @m = ();
  if ($i % 3 == 0) {
    push(@m, 'fizz');
  }
  if ($i % 5 == 0) {
    push(@m, 'buzz');
  }
  print((@m ? join('', @m) : $i) . "\n");
}

File: nodejs/fizzbuzz.js
Array.from(new Array(30).keys(), (n) => {
  const v = n + 1;
  const m = [];
  if (v % 3 === 0) {
    m.push('fizz');
  }
  if (v % 5 === 0) {
    m.push('buzz');
  }
  console.log(m.length ? m.join('') : v.toString());
});
